src/main/java/com/teamacronymcoders/base/materialsystem/blocks/SubBlockOreSamplePart.java

```diff
@@ -185,12 +194,12 @@ public boolean canPlaceBlockAt(World world, @Nonnull BlockPos pos) {
     public boolean onBlockActivated(World world, BlockPos pos, EntityPlayer player) {
         if (activatedText != null && !activatedText.isEmpty()) {
             player.sendStatusMessage(new TextComponentString(activatedText), true);
-        } else {
+            world.setBlockToAir(pos);
+        } else if (!requireTool) {
             spawnItemStackEntity(world, this.getItemStack().copy(), pos);

```

Comment:
While poking around in here, is there a chance you could also take a quick look at replacing spawnItemStackEntity, with the method mentioned here: https://github.com/The-Acronym-Coders/BASE/issues/93